spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,131 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+  describe "create" do
+    let(:user) {User.create(:first_name => "Rebecca", :last_name => "Holzschuh", :email => "rebeccaholzschuh@gmail.com", :username=> "rebecca")}
+    it "adds one user to the database" do
+      expect {user}.to change(User, :count).by(1)
+    end
+
+    it "adds the user to the database" do
+      User.create(:first_name => "Rebecca")
+      expect(User.last.first_name).to eq("Rebecca")
+    end
+
+    it "redirects back to the home page after saving a user" do
+      post :create, user: {:first_name => "Rebecca", :last_name => "Holzschuh"}
+      expect(response).to redirect_to root_path
+    end
+  end
+
+  before(:each) do
```

Comment:
Putting this at the highest level, you are making it for the test above as well, which unexpectedly adds a user to your first test.